mavutil.py

```diff
@@ -397,9 +397,9 @@ def param_fetch_one(self, name):
         '''initiate fetch of one parameter'''
         try:
             idx = int(name)
-            self.mav.param_request_read_send(self.target_system, self.target_component, "", idx)
+            self.mav.param_request_read_send(self.target_system, self.target_component, b'', idx)
         except Exception:
-            self.mav.param_request_read_send(self.target_system, self.target_component, name, -1)
+            self.mav.param_request_read_send(self.target_system, self.target_component, name.encode('utf8'), -1)
```

Comment:
Shouldn't `param_fetch_one()` really take a Python string?

Reply:
Yes, it does — for this reason we need to cast it to a binary string before passing it to param_request_read_send.